test/fuzz/README.md

```diff
@@ -0,0 +1,37 @@
+# libFuzzer tests
+
+## Setup
+
+Build r2 with sanitizers
+
+```shell
+# Linux
+export CC=clang-14
+# macOS
+export CC="$(brew --prefix llvm@14)/bin/clang"
+
+export CFLAGS="-g"
+rm -rf build
+python3 ./sys/meson.py --sanitize fuzzer,address --install
+```
+
+Build fuzzers
+
+```shell
+cd test/fuzz
+mkdir build
+$CC \
+  -fsanitize=fuzzer,address \
+  -isystem "../../libr/include" \
+  -isystem "../../shlr/sdb/src" \
+  ../../build/libr/socket/libr_socket.a \
+  ../../build/libr/util/libr_util.a \
+  -o build/fuzz_r_run_parseline fuzz_r_run_parseline.c
```

Comment:
Would appreciate any help getting this wired up with meson

Reply:
i think its fine to just use makes for the fuzzing testsuite for now

Reply:
Done with Meson!